src/notte/llms/prompts/extract-json-schema/multi-entity/user.md

```diff
@@ -0,0 +1 @@
+Transformed content:
```

Comment:
Suggestion, maybe just adding a action / verb would yield better results. Your turn, provide me with the transformed content" or something like that